framework/src/main/aidl/org/eu/droid_ng/wellbeing/lib/IWellbeingFrameworkService.aidl

```diff
@@ -0,0 +1,10 @@
+// IWellbeingFrameworkService.aidl
+package org.eu.droid_ng.wellbeing.lib;
```

Comment:
lib -> framework

Reply:
Changing name of this would also require changing the name of:
[`app/src/main/aidl/org/eu/droid_ng/wellbeing/lib/IWellbeingFrameworkService.aidl`](https://github.com/Fox2Code/android_packages_apps_Wellbeing/blob/master/app/src/main/aidl/org/eu/droid_ng/wellbeing/lib/IWellbeingFrameworkService.aidl)

They are supposed to be identical copies to each other, I duplicated the files to avoid to complexify the build script.
Both files having the exact same name is a hard requirement of the aidl protocol.

Reply:
:+1: